actions/publish-image/action.yaml

```diff
@@ -203,12 +203,23 @@ runs:
 
   - name: Attest provenance
     shell: bash
-    run: |        
-      if slsactl download provenance --format=slsav1 "${IMG_NAME}" > provenance-slsav1.json; then
-        cat provenance-slsav1.json
-        cosign attest --yes --predicate provenance-slsav1.json --type slsaprovenance1 "${IMG_NAME}"
-      else
-        slsactl download provenance --format=slsav1 "${IMG_NAME}"
-        echo "ERROR: Failed to generate slsav1 provenance. Check whether the image is present in the Prime registry."
-        exit 3
-      fi
+    if: ${{ inputs.push-to-prime == true || inputs.push-to-prime == 'true' }}
+    run: |
+      max_retries=3
```

Comment:
Can you surround your vars with curly braces?

Reply:
+1

Ideally it would also be double quoted.